LEGO1/lego/legoomni/src/common/legoanimationmanager.cpp

```diff
@@ -1,7 +1,17 @@
 #include "legoanimationmanager.h"
 
+#include "legogamestate.h"
+#include "legoomni.h"
+#include "misc.h"
+#include "mxutilities.h"
+
+#include <io.h>
+
 DECOMP_SIZE_ASSERT(LegoAnimationManager, 0x500)
 
+// GLOBAL: LEGO1 0x100f7048
+Character g_characters[0x2f];
```

Comment:
Might be cleaner to represent the declaration of the size in decimal?

Reply:
Ultimately it's a preinitialized array, but I changed it for now.